v2/app/src/main/java/com/google/firebase/example/makeitso/data/repository/TodoListRepository.kt

```diff
@@ -0,0 +1,28 @@
+package com.google.firebase.example.makeitso.data.repository
+
+import com.google.firebase.example.makeitso.data.datasource.TodoListRemoteDataSource
+import com.google.firebase.example.makeitso.data.model.TodoList
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+class TodoListRepository @Inject constructor(
+    private val todoListRemoteDataSource: TodoListRemoteDataSource
+) {
+    val todoLists: Flow<List<TodoList>> = todoListRemoteDataSource.getTodoLists("")
```

Comment:
Is this parameter supposed to be an empty string?

Reply:
No, I'll add this after adding the UI - this ID will be passed when navigating to this screen, and the "onClick" function is not yet implemented.